static/js/transform-facets.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * A Facets component transformFacets function which applies the configuration
+ * specified by the Facets 'fields' option
+ * 
+ * @param {DisplayableFacet[]} facets from answers-core
+ * @param {FilterOptionsConfig} config from answers-search-ui
+ * @returns {DisplayableFacet[]} from answers-core
```

Comment:
Is the return type a union of `DisplayableFacet` and `FilterOptionsConfig`?

Reply:
ah, ya, thanks for catching that